### DISCUSSION TITLE
Possibility to provide the .efi (and .bin?) as separate download?

### DISCUSSION THREAD

**Discussion by MacGyverNL** [memtest86plus/memtest86plus#190] [Enhancement]
I like adding memtest86(+) to the EFI system partition of my systems, so I can run memtests without needing to USB- or PXE-boot (I have a very obstinate mainboard which can't easily boot from USB).

Right now, as far as I can tell, I can either build from source, or download the iso, loopback-mount the contained partition, and copy the EFI binary from there. Since it's being built anyway, would it be possible to provide the EFI binary that's in the release-version ISO as a separate standalone download (possibly gated behind an archive link for advanced users)?

**Comment by x86fr** [comment]
You can find them here: https://memtest.org/download/v6.00/mt86plus_6.00.binaries.zip

**Reply by MacGyverNL** [reply]
Thanks!
